air/tests/test_module/negative_tests/farewell_step.rs

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use air::FarewellError;
+use air_test_utils::prelude::*;
+
+#[test]
+fn callresult_are_not_empty() {
+    let air = r#"(null)"#;
+    let client_peer_id = "some_peer_id";
+    let mut client_vm = create_avm(unit_call_service(), client_peer_id);
+
+    let prev_data = InterpreterData {
+        versions: Versions::new(semver::Version::new(1, 1, 1)),
+        trace: <_>::default(),
+        global_streams: <_>::default(),
+        restricted_streams: <_>::default(),
+        last_call_request_id: 0,
+        cid_info: <_>::default(),
+    };
+    let prev_data: Vec<u8> = serde_json::to_vec(&prev_data).unwrap();
+    let mut call_results_4_call = CallResults::new();
+    let mut call_results_4_err = air_interpreter_interface::CallResults::new();
```

Comment:
let's rename it to `expected_call_results_error` and move closer to place of usage